torch/csrc/jit/ir.h

```diff
@@ -227,6 +227,8 @@ struct Value {
   //          %5 = h(%6, %6)
   TORCH_API void replaceAllUsesWith(Value* newValue);
 
+  TORCH_API void replaceAllUsesAfterNodeWith(const Node* node, Value* newValue);
```

Comment:
actually, comments should be added here.

Reply:
Thanks Lu, added there